CMakeLists.txt

```diff
@@ -180,6 +181,7 @@ set (RenderManager_PUBLIC_HEADERS
 	"${CMAKE_CURRENT_BINARY_DIR}/osvr/RenderKit/Export.h"
 )
 add_library(osvrRenderManager ${RenderManager_SOURCES} ${RenderManager_PUBLIC_HEADERS})
+target_compile_features(osvrRenderManager PRIVATE cxx_range_for)

```

Comment:
I forgot about this line, it forces CMake to add compiler flags for C++11. I think it's useful to keep it in this PR, since OSVR relies on C++11.